integration/run_test.go

```diff
@@ -157,6 +167,45 @@ func TestRun(t *testing.T) {
 	}
 }
 
+func TestRunTail(t *testing.T) {
+	MarkIntegrationTest(t, CanRunWithoutGcp)
+
+	for _, test := range tests {
+		t.Run(test.description, func(t *testing.T) {
+			if test.targetLog == "" {
+				t.SkipNow()
+			}
+			ns, _ := SetupNamespace(t)
+
+			args := append(test.args, "--tail")
+			out := skaffold.Run(args...).InDir(test.dir).InNs(ns.Name).WithEnv(test.env).RunLive(t)
+
+			WaitForLogs(t, out, test.targetLog)
+
+			skaffold.Delete().InDir(test.dir).InNs(ns.Name).WithEnv(test.env).RunOrFail(t)
+		})
+	}
+}
+
+func TestRunTailDefaultNamespace(t *testing.T) {
+	MarkIntegrationTest(t, CanRunWithoutGcp)
+
+	for _, test := range tests {
+		t.Run(test.description, func(t *testing.T) {
+			if test.targetLog == "" {
+				t.SkipNow()
+			}
+
+			args := append(test.args, "--tail")
+			out := skaffold.Run(args...).InDir(test.dir).WithEnv(test.env).RunLive(t)
+
+			WaitForLogs(t, out, test.targetLog)
+
+			skaffold.Delete().InDir(test.dir).WithEnv(test.env).RunOrFail(t)
```

Comment:
Is running all test in default ns increasing the test run time? Is yes, by how much?

Reply:
this will run against only tests which have `targetLog` defined, right now that's 4 tests. so it'll increase the run time by however long it takes to run 4 more tests, twice (since we're doing it for default/non-default namespaces).